bigbone/src/main/kotlin/social/bigbone/api/method/MediaMethods.kt

```diff
@@ -15,31 +16,177 @@ import java.io.File
  * @see <a href="https://docs.joinmastodon.org/methods/media/">Mastodon media API methods</a>
  */
 class MediaMethods(private val client: MastodonClient) {
+
+    private val endpoint: String = "api/v1/media"
```

Comment:
Not entirely true: I only added the `V1` and `V2` suffix if there’s more than one. Otherwise, the version is implied to be the newest if all share the same.

Reply:
✅